### ISSUE TITLE
sysctl start beacon.service panics on startup

### ISSUE THREAD

**Issue by tmcgukin** [prysmaticlabs/prysm#7237]
I was running my services for over two weeks(whenever sync was fixed) with zero issues and then my beacon just stopped working.  When restarting my beacon service it goes through about 10 seconds of the loading and errors out and restarts.

Error : level=fatal msg="Could not cache justified state balances: could not get ancestor state: unknown block" prefix=blockchain

Switching to the ./prysm.sh I am able to run the beacon with no issues(note there is a lot of this ERROR pool/attestations: Cannot delete seen attestations error=<nil>) but still syncing from 74k up to current.  

My service tags are as follows --p2p-host-ip=${ClientIP} --datadir=/var/lib/prysm/beaconchain --http-web3provider=http://127.0.0.1:8545/' --p2p-max-peers=60 --dev --block-batch-limit=512

I have since removed the ones added during the syncing issue maxing etc.  But still getting the same error as when the beacon chain first stopped. 

Running Alpha 25 now with the issue

**Comment by prestonvanloon** [comment]
The error "Cannot delete seen attestations" is known: #7236.

**Comment by terencechain** [comment]
Double checked, the `"Could not cache justified state balances: could not get ancestor state: unknown block"`error is just an older version without the following fix

https://github.com/prysmaticlabs/prysm/pull/6861

As long as you keep running Alpha 25, it'll be fine. "Cannot delete seen attestations" can be ignored

**Comment by terencechain** [comment]
Going to close this unless `Could not cache justified state balances: could not get ancestor state: unknown block` happens in alpha25

**Comment by tmcgukin** [comment]
> Double checked, the `"Could not cache justified state balances: could not get ancestor state: unknown block"`error is just an older version without the following fix
> 
> #6861
> 
> As long as you keep running Alpha 25, it'll be fine. "Cannot delete seen attestations" can be ignored

Not too familiar with github, I looked through 6861, but didnt see anything clear.  What exactly should be changed on my side?

**Comment by prestonvanloon** [comment]
@terencechain this user report comes from alpha.25.

**Comment by terencechain** [comment]
@prestonvanloon got it, i was under the impression `Could not cache justified state balances` came from versions before 25. Either way, can't reproduce this and haven't seen this in 24/25

**Comment by terencechain** [comment]
Just chatted with @tmcgukin. The script is working for 25. The other setup where it uses `sysctl start beacon.service` doesn't work. Asked to check operational setup

![image](https://user-images.githubusercontent.com/21316537/93132121-fafc1b80-f689-11ea-8a52-7fa9f65df6d9.png)

**Comment by terencechain** [comment]
Thanks @dv8silencer for the DB. I looked into this. One of the parent block `be5d14a2329be5e3cbb263ed82a7729ec06dca90c138bcf9dd3be3d2596b23a4` is missing in the DB. The question is how did it get into this state? and what we can do to prevent beacon node getting into this state? I'll open a preventive PR on this this week

**Comment by terencechain** [comment]
there's no more actions to follow up with this. I think this can be closed after the merging of #7245
If it happens again, please reopen this and we can take a look

**Comment by cin3rama** [comment]
I needed to upgrade our test server's disk drive as it was beyond 90% full.  Prior to starting the process I deleted some old files, namely the old prysm v1.X.X files up to but not including v1.3.10 and some extraneous tar.gz files, to open up some space.  In addition, I toggled the --historical-slasher-node from TRUE to FALSE.  I did this as a test to learn what I might encounter on the production server when the slasher moves to the beacon node in the next release.  I did not think that I'd draw a fatal or unrecoverable error .  Here is what I found:

```
prysm.sh[23271]: time="2021-06-24 16:42:37" level=info msg="Connected to eth1 proof-of-work chain" endpoint="http://127.0.0.1:8545" prefix=powchain
prysm.sh[23271]: time="2021-06-24 16:42:37" level=info msg="Blockchain data already exists in DB, initializing..." prefix=blockchain
prysm.sh[23271]: time="2021-06-24 16:42:38" level=info msg="Chain genesis time reached" genesisStateRoot=2bb257ca66d05a047a65fe43a5f457b674de445d917cca029efb09b3ba4758c4 genesisTime="2020-11>
prysm.sh[23271]: time="2021-06-24 16:42:38" level=fatal msg="Could not cache justified state balances: could not get ancestor state: unknown block" prefix=blockchain
systemd[1]: beacon-chain.service: Main process exited, code=exited, status=1/FAILURE
systemd[1]: beacon-chain.service: Failed with result 'exit-code'.
```
After several attempts to resolve the error including, upgrading the disk drive size, upgrading to Prysm v1.3.11 from v1.3.10 and adjusting the rpc settings to default in the both the beacon-chain and validator config.yaml files the problem persists.   Below are the configs for the beacon-chain and validator.

1. beacon-chain, validator and eth1(geth) exist on the same server
2. beacon-chain config - historical-slasher-node: true disable-broadcast-slashings: true pyrmont: true http-web3provider: http://localhost:8545
3. validator config - wallet-dir:  "server/directory" pyrmont: true graffiti: "graffiti"
4. Server is an ARM64 (4core/16gb) running ubunut 20.04, geth v1.10.4, prysm v1.3.11

Any help in resolving this issue would be greatly appreciated.
